std/bigint.d

```diff
@@ -540,7 +540,8 @@ public:
     /**
         Implements BigInt unary operators.
      */
-    BigInt opUnary(string op)() pure nothrow const if (op=="+" || op=="-" || op=="~")
+    BigInt opUnary(string op)() pure nothrow const @safe
```

Comment:
You're right, I forgot to take that out.  I only put it in so that the compiler will do the tedious work of figuring out what exactly is making it `@system` for me.